lens/base/view-set.rkt

```diff
@@ -6,10 +6,11 @@
 (module+ test
   (require rackunit))
 
-(provide lens-view
-         lens-set
-         lens-view*
-         lens-set*)
+(provide
+ lens-view*
+ lens-set*
```

Comment:
It's a shame there isn't a list*/c combinator provided out of the box (unless there is in 6.2, I haven't upgraded on my machine yet)